### PR TITLE
Created Service Result class and ProblemTypes class

### PR DESCRIPTION
**Pull request by jonathan-osorio**
Created Service Result class and ProblemTypes class

### CODE REVIEW

File: RealestateApi/Service/ServiceResult.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿namespace RealEstateApi.Service
+{
+    public class ServiceResult<T>
+    {
+        public bool IsSuccess { get; set; }
+        public string ProblemTypes { get; set; }
```
Comment (Silva096) [inline RealestateApi/Service/ServiceResult.cs:6]: ProblemType :)

File: RealestateApi/Service/ServiceResult.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        public string AdditionalInformation { get; set; }
+        public T Result { get; set; }
+
```
Comment (Silva096) [inline RealestateApi/Service/ServiceResult.cs:9]: extra line

File: RealestateApi/Service/ServiceResult.cs (hunk 1 above)
Comment (jonathan-osorio) [reply RealestateApi/Service/ServiceResult.cs]: Done

File: RealestateApi/Service/ServiceResult.cs (hunk 2 above)
Comment (jonathan-osorio) [reply RealestateApi/Service/ServiceResult.cs]: Done

**Review by diogo-s4ntos — APPROVED** [review]

**Review by RafaelMeha — APPROVED** [review]

**Pull request merged**